compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -1883,10 +1883,7 @@ void addImportPackageDeclaration(DiagnosticPos pos,
                                      List<String> nameComps,
                                      String version,
                                      String alias) {
-        // Disabling versioned import until compiler support semvar versioned imports
-        if (!(version == null || version.isEmpty())) {
-            this.dlog.error(pos, DiagnosticCode.VERSIONED_IMPORT_NOT_SUPPORTED);
```

Comment:
Shall we remove `VERSIONED_IMPORT_NOT_SUPPORTED` from `DiagnosticCode.java` if it is no longer used?
Also from `compiler.properties`

Reply:
+1

Reply:
Fixed.